src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -1327,6 +1327,55 @@ def getConfigCacheIDs(self):
                 IDs.append(ID)
         return IDs
 
+    def physicsTaskType(self):
+        """
+        _physicsTaskType_
+
+        Return the physics Task type
+        """
+        return getattr(self.data, 'physicsTaskType', None)
+
+    def getStepPhysicsTypes(self):
```

Comment:
If I fail to mention it in the final review description, given that this information is for monitoring purpose only. Do we really need to define the physics type at the CMSSW step level?

My comment goes along the lines of the campaign discussion we had yesterday. Perhaps for StepChain, we could set it at the WMTask level with a comma separated string?

Reply:
I already mentioned this in the general comment, but in short, we don't really need to define it, but since it is defined already, do we really want to remove it? Specially if this is supported by cmssw at long term, having it as a step property doesn't seem like a bad idea for this particular case, even if we only use it for monit at this point.